README.md

```diff
@@ -351,6 +351,52 @@ Auth0
 
 Find out more about [Setting up Facebook Login](https://auth0.com/docs/connections/nativesocial/facebook) with Auth0.
 
+### Organizations
+
```

Comment:
Let's add in a little more context around Organizations here as to what it is, and later we can link to the docs on this when they are published.

In an attempt to avoid writing new material around this and utilising what has already been devised by our content writers, we could just go with the intro text on [the beta docs](https://auth0content-pr-9507.herokuapp.com/docs/organizations), which is:

> The Organizations feature represents a broad update to the Auth0 platform that allows our Business-to-Business (B2B) customers to better manage their partners and customers, and customize the ways that end-users in those organizations access their applications. Auth0 customers can use Organizations to:
> * represent their business customers and partners in Auth0 and manage their membership.
> * configure branded, federated login flows for each business.
> * build administration capabilities into their products, using Organizations APIs, so that those businesses can manage their own organizations.

Then we can add a link into the docs here later when they're published. What do you think? Whatever we decide here we can put as standard on the other SDK readme's too.

Reply:
Hold off on this change for a bit though, I've asked for clarification with the team internally.

Reply:
Done, thanks.